### PR TITLE
EI-3331 - chore: Bump cryptography to 43.0.1

### PR DESCRIPTION
**Pull request by vtermanis**
Sanity performance results attached to ticket - looks OK.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -18,7 +18,7 @@ max-complexity = 15
 
 [metadata]
 name = iotics-identity
-version = 2.1.1
+version = 2.1.2
```
Comment (vtermanis) [inline setup.cfg:21]: See commit comment regarding reasoning around patch vs minor increment. Also:
- https://github.com/pyca/cryptography/blob/main/CHANGELOG.rst#4300---2024-07-20
- https://mta.openssl.org/pipermail/openssl-announce/2020-March/000167.html

File: README.md [hunk 2]
```diff
@@ -1,9 +1,8 @@
 # iotics-identity-py
 
-[![PyPi version](https://pypip.in/v/iotics-identity/badge.svg)](https://pypi.org/project/iotics-identity/)
```
Comment (vtermanis) [inline README.md:3]: These no longer worked

**Comment by conbon** [comment]
any chance you could make a slight update to the readme instruction for `pip install .[dev]` to `pip install .'[dev]'`?

**Review by conbon — APPROVED** [review]
See comment - `zsh` shell problems on mac again. Common enough for users to hit. 
If the same works on linux it's worth updating I think.

**Comment by vtermanis** [comment]
@conbon  c186e5d10

**Pull request merged**